### ISSUE TITLE
Exercise 5 Level 6: Too many non-private methods

### ISSUE THREAD

**Issue by jingyiiiiz** [nus-cs2030/2223-s1#181]
After reaching Level 6 of Exercise 5, I got this error when uploading my answers to Codecrunch to compile: 
```
Too many non-private methods.
Grading terminated.
```

All I have written are the methods required in the question, and one more getThing() method to get this.thing. May I ask if there's a way to work around this error? Do I need to change the access of any of my methods? Thanks in advance!

**Comment by MingFengC** [comment]
Hi! For my solution, I did not use any form of getters

**Comment by jingyiiiiz** [comment]
Hi @MingFengC, thanks for your reply! Then I'm wondering how to return the Maybes needed, since if we directly return supplier.get() or mapper.apply(this.thing) it will result in incompatible type errors. (My previous attempt was using getters to extract out the values first then put into new Maybes)

**Comment by MingFengC** [comment]
Yes, I faced this issue initially as well. You can use previous methods defined to extract out the value of `Maybe.thing` and return the new Maybes instead

**Comment by wilsonwid** [comment]
I agree with @MingFengC. Level 6 (and a part of level 5, actually) requires you to use other methods that you've defined previously instead of using getter(s).
